.storybook/FormField.js

```diff
@@ -179,6 +188,72 @@ storiesOf('InputField', module)
       </Flex>
     ))
   )
+  .add(
+    'FormField composed with Select',
+    withInfo(
+      'This example demonstrates how the input field will behave with a select dropdown'
+    )(() => (
+      <Flex width={900}>
+        <Box width={1 / 3} pl={3}>
+          <Text bold pt={4} pb={2}>
+            User Types a Value
+          </Text>
+          <FormField onChange={() => {}}>
+            <Label>Cabin Class</Label>
+            <Icon name="key" color="blue" size="18" />
+            <Select
+              id="cabinClass"
+              name="cabinClass"
+              defaultValue="Premium Economy"
+              prefixIcon="key"
+            >
+              <option>Economy</option>
+              <option>Premium Economy</option>
+              <option>Business</option>
+              <option>First Class</option>
+              <option>
+                With a super long label that doesn't get clobbered by the
+                chevron
+              </option>
+            </Select>
+          </FormField>
+        </Box>
+      </Flex>
+    ))
+  )
+  .add(
```

Comment:
Nice to see your PR here. I wonder what these red colored lines indicate in the code diff. Does anyone know? (Never saw before in previous PR reviews)

Reply:
I'm not a 100% sure here but maybe its a change in whitespace from my ide...